package.json

```diff
@@ -84,17 +91,20 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
+    "@ava/get-port": "^2.0.0",
```

Comment:
should this be a peer?

I wish npm had something like cargo features 😭

Reply:
> I wish npm had something like cargo features 😭

i know, right

Reply:
i guess technically this should be multiple packages, with `edgespec/ava` being one of them @codetheweb

Reply:
yeah, that works since the headless dev server is a public API...
I really don't feel like setting up a mono repo rn haha so I'll make it a peer for now and add it to the docs